xml/deployment_cifs.xml

```diff
@@ -0,0 +1,290 @@
+   <dm:status>editing</dm:status>
+   <dm:deadline></dm:deadline>
+   <dm:priority></dm:priority>
+   <dm:translation></dm:translation>
+   <dm:languages></dm:languages>
+   <dm:release>SES4</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+  This section describes how to export &cephfs; via Samba share. Samba shares
+  can be used with Windows* clients.
+ </para>
+ <warning>
+  <title>Technology Preview</title>
+  <para>
+   As of &storage; 5, <literal>CIFS Export</literal> is considered a technology
+   preview and is not supported.
+  </para>
+ </warning>
+ <sect1 xml:id="sec.ses.cifs.example">
+  <title>Example Installation</title>
+  <para>
+   Exporting &cephfs; is a technology preview and not supported. To enable
+   CIFS, you have to manually install Samba on one cluster node and configure it.
+   Failover functionality can be provided with CTDB and the &sle; &hasi;.
+  </para>
+  <para>
+   First install the Samba daemon on one node:
+  </para>
+<screen>&prompt.root;<command>zypper</command> in samba</screen>
+  <para>
+   Then edit the <filename>/etc/smb/smb.conf</filename> and add the following
+   section:
+  </para>
+<screen>[<replaceable>SHARE_NAME</replaceable>]
+ceph:config_file = /etc/ceph/ceph.conf</screen>
+  <para>
+   Start and enable the Samba daemon:
+  </para>
+<screen>&prompt.root;<command>systemctl</command> start smb.service
+&prompt.root;<command>systemctl</command> enable smb.service</screen>
+ </sect1>
+ <sect1 xml:id="sec.ses.cifs.ha">
+  <title>High Availability Active-Passive Configuration</title>
+  <para>
+   This section provides an example of how to set up a two-node active-passive
+   configuration of Samba servers. The setup requires the &sle; &hasi;.
+   The 2 nodes are called &si.earth; and &si.mars;.
+  </para>
+  <para>
+   For details about &sle; &hasi;, see
+   <link xlink:href="https://www.suse.com/documentation/sle-ha-12/" />.
+  </para>
+  <para>
+   In this setup &si.earth; has the
+   IP address <systemitem class="ipaddress">192.168.1.1</systemitem> and
+   &si.mars; has the address
+   <systemitem class="ipaddress">192.168.1.2</systemitem>.
+  </para>
+  <para>
+   Additionally 2 floating virtual IP addresses are used, allowing clients
+   to connect to the service independent of which physical node it is running on.
+   <systemitem class="ipaddress">192.168.1.10</systemitem> is used for cluster
+   administration with Hawk2 and <systemitem class="ipaddress">192.168.2.1</systemitem>
+   is used exclusively for the CIFS exports. This makes it easier to apply
+   security restrictions later.
+  </para>
+  <para>
+   The following procedure describes the example installation. More details
+   can be found at <link xlink:href=
+   "https://www.suse.com/documentation/sle-ha-12/install-quick/data/install-quick.html" />.
+  </para>
+  <procedure xml:id="proc.sec.ses.cifs.ha">
+   <step>
+    <para>
+     Prepare the nodes used to host the Samba service:
```

Comment:
It's not clear on which nodes you need to execute this. So maybe rephrase it like "On both nodes, prepare the Samba service(s):"